src/tools/gdb/plan_dump.py

```diff
@@ -0,0 +1,611 @@
+# To setup the command inside gdb session, run:
+#    source path_to_gpdb_sources/src/tools/gdb/plan_dump.py
+# plan_dump_cmd is a gdb command which may be useful at core-dump debugging
+# process. This command prints the plan tree, like the EXPLAIN does,
+# but the command is less informative (also command works with Plan* structures
```

Comment:
maybe not "also", but "because"?

Reply:
No, this is not the root cause of less informative output (it is an additional information that shows difference of script with original explain). 
The output of script is less informative because of:
- the script is supposed to be used on the core-dumps, so some information like index names or partitions names may be hard (or unreal) to get, because of abscence of cache (or original tables)
- also this is an initial version of the command, there is no implemented functionality for showing qualifiers or, for ex. index condition